tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -76,28 +77,16 @@ func setupAWSCloudProvider(t *testing.T) {
 	// The e2e test uses Microsoft Windows Server 2019 Base with Containers image, m4.large type, and libra ssh key.
 	cloud, err := cloudprovider.CloudProviderFactory(kubeconfig, awscredentials, "default", dir,
 		imageID, instanceType, sshKey, privateKeyPath)
-	if err != nil {
-		t.Fatalf("Error obtaining aws interface object %v", err)
-	}
+	require.NoError(t, err, "Error obtaining aws interface object")
```

Comment:
I think, the error messages might be suppressed, please do `NoErrorf`

Reply:
Comments apply everywhere

Reply:
done.

Reply:
@vinaykns please confirm that `NoError` does not post the error message when it encounters a failure.

Reply:
NoError does post the error message when it encounters a failure, but message wont be concatenated.
for ex: http://pastebin.test.redhat.com/816049 . If we want the message to be concatenated with the error, then we can use NoErrorf else we can go with NoError

Reply:
My vote is to stick with `NoError` then

Reply:
Yeah, I am fine as well then :)